src/components/DataTable/DataTable-story.js

```diff
@@ -19,69 +18,54 @@ const props = () => ({
 
 storiesOf('DataTable', module)
   .addDecorator(withKnobs)
-  .add(
-    'default',
-    withReadme(readme, () => require('./stories/default').default(props())),
-    {
-      info: {
-        /* eslint-disable no-useless-escape */
-        text: `
+  .addParameters({ notes })
+  .add('default', () => require('./stories/default').default(props()), {
```

Comment:
Cannot see the README for data table at https://5c7fd6f20bc108000b085479--carbon-components-react.netlify.com/experimental/?path=/story/datatable--default, unless I'm missing something?

Reply:
Hmmm not sure which preview that link is from but the latest preview has it working https://deploy-preview-1972--carbon-components-react.netlify.com/

The dark theme is currently not using themes properly, I have an open issue with SB.

Reply:
OK I can see the README now, thanks @vpicone! I see very low contrast there (black text in black background), but I think it's something you are fixing - Is it correct?

Reply:
Yes, storybook 5 allows for theming of addons as well. I have a PR to enable that feature for the notes addon: https://github.com/storybooks/storybook/pull/5922#event-2187964702